README.md

```diff
@@ -3,29 +3,21 @@
 This repository is meant to help new users start using the Arm-based AWS Graviton and Graviton2 processors which power the latest generation of Amazon EC2 instances. While it calls out specific features of the Graviton processors themselves, this repository is also generically useful for anyone running code on Arm.
 
 # Contents
-* [Transitioning to Graviton](#transitioning-to-graviton)
-* [Building for Graviton](#building-for-graviton-and-graviton2)
-* [Optimizing for Graviton](optimizing.md)
-* [Taking advantage of Arm Advanced SIMD instructions](SIMD_and_vectorization.md)
-* [Recent software updates relevant to Graviton](#recent-software-updates-relevant-to-graviton)
-* Language-specific considerations
-	* [C/C++](c-c++.md)
-	* [Go](golang.md)
-	* [Java](java.md)
-	* [.NET](dotnet.md)
-	* [Python](python.md)
```

Comment:
this chase seems to have done more than just add a Lambda page?

Reply:
This adds a Lambda page as well as some code samples to show customers how to migrate their functions and how to test Lambda on x86 and arm. Customers can use the code to deploy a sample Lambda function and compare performance.

Reply:
My point is that the change significantly changed the table of contents beyond adding the lambda section.

Reply:
Ah, I see, yes, this shouldn't have been as big a change to the TOC, I think I've been out of sync with other updates. I'll close PR and reopen with a cleaner one.